samples/drivers/flash_sim/prj.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_STDOUT_CONSOLE=y
+
+CONFIG_FLASH=y
+CONFIG_FLASH_SIMULATOR=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
```

Comment:
I expect it is not require for the emulation driver enabling real flash writing at all.